public/index.html

```diff
@@ -24,6 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Bitter|Montserrat&display=swap"
```

Comment:
You can optimize this by defining the required don't weights. It's not required, just fyi ;-)